crates/tests-e2e/src/coordinator.rs

```diff
@@ -23,18 +23,19 @@ pub struct Coordinator {
 }
 
 #[derive(Deserialize)]
-pub struct DlcChannels {
+pub struct SubChannels {
```

Comment:
What am I missing here? Why did you change this to `SubChannels` from `DlcChannels`? 
Might have been a rebase error.

Reply:
rebase error indeed!